README.md

```diff
@@ -33,3 +33,21 @@ In the pull request, please make it clear how we can verify that the contributio
 This could be accomplished with a link to a demonstration experiment, the inclusion of an example file and/or testing files, or through some other means.
 We try to review pull requests quickly and add new contributions as soon as the minimal standards are met.
 
+## Plugin templates
+
+There are two plugin template directories inside the `/packages` directory that you can as a reference when creating a directory for your plugin contribution. Both templates are compatible with jsPsych v7+.
+
+* `plugin-template-ts`: This template uses TypeScript source files that are complied into JavaScript using Node.js and npm. This is the format used for plugins in the main jsPsych repo. For more details, please see the jsPsych documentation page [Configuring the jsPsych development environment](https://www.jspsych.org/developers/configuration).
+* `plugin-template`: This template allows you to put your plugin's JavaScript code directly into a JavaScript template file, rather than using TypeScript and Node.js/npm. 
+
+## jsPsych version compatibility
+
+We would like to encourage you to contribute plugins that are compatible with the latest jsPsych version. At the same time, we realize that there may be jsPsych users who have created very useful plugins with jsPsych v6 that they would like to share with the community, but don't have the time/resources to convert into the jsPsych v7+ Node package format. Therefore we welcome plugins that are compatible with v6 as well as v7+. 
+
+If you'd like to contribute a jsPsych v6 plugin, please do the following:
+* Use the `plugin-template` directory as a reference
+* Delete everything inside of the `index.js` template file and replace it with your v6-compatible plugin code
+* Edit the `package.json` file with the information about your plugin
+* Delete the "devDependencies" property and values in the `package.json` files (this is only relevant for plugins that are compatible with jsPsych v7+)
```

Comment:
Since `@jspsych/config` contains babel, it should technically be specified as a dev dependency (while it would still work without that in the monorepo because other packages depend on `@jspsych/config` too, so it's already installed there).
```suggestion
* Change the "jspsych" version field in "devDependencies" in `package.json` to "^6.3.1"
```
I'll push the previous jsPsych releases to NPM today so that will work then.